internal/controller/management_controller.go

```diff
@@ -616,21 +623,48 @@ func applyKCMDefaults(config *apiextensionsv1.JSON) (*apiextensionsv1.JSON, erro
 		}
 	}
 
-	// Those are only needed for the initial installation
-	enforcedValues := map[string]any{
+	chartutil.CoalesceTables(values, defaultValues)
+	raw, err := json.Marshal(values)
+	if err != nil {
+		return nil, err
+	}
+	return &apiextensionsv1.JSON{Raw: raw}, nil
+}
+
+func applySveltosDefaults(config *apiextensionsv1.JSON) (*apiextensionsv1.JSON, error) {
+	result, err := applyDefaultsToConfig(config, map[string]any{
+		"projectsveltos": map[string]any{
+			"registerMgmtClusterJob": map[string]any{
+				"registerMgmtCluster": map[string]any{
+					"args": []string{
+						// expected to be in format: --labels=labelA=A,labelB=B,labelC=C
+						"--labels=" + K0rdentManagementClusterLabelKey + "=" + K0rdentMManagementClusterLabelValue + "",
```

Comment:
```suggestion
						"--labels=" + K0rdentManagementClusterLabelKey + "=" + K0rdentMManagementClusterLabelValue,
```

And do we need these constants globally? If so, do we need them (probably just the key) in the public API?